docs/tutorials/cmip6-recipe.ipynb

```diff
@@ -62,25 +62,25 @@
      "output_type": "stream",
```

Comment:
<p>The correct way to link to another page is not via a direct url but a myst markdown directive. I believe this should be {doc}<code>file_patterns</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
:sweat_smile: Actually could not figure out how to do this, so I just removed all of the docs urls, in case you want to merge over the weekend. We can revisit links Monday if desired (I should learn how to do this eventually), but actually I think this notebook reads fine without them.